src/lib.rs

```diff
@@ -89,11 +89,11 @@ pub fn remove_deb_temp_directory(options: &Config) {
     let _ = fs::remove_dir(&deb_temp_dir);
 }
 
-/// Builds a release binary with `cargo build --release`
+/// Builds a binary with `cargo build`
 pub fn cargo_build(options: &Config, target: Option<&str>, other_flags: &[String], verbose: bool) -> CDResult<()> {
     let mut cmd = Command::new("cargo");
     cmd.current_dir(&options.manifest_dir);
-    cmd.arg("build").args(&["--release", "--all"]);
```

Comment:
This looks like a breaking change. We need `--all` to allow packaging multiple binaries/libraries from a workspace.

Reply:
sure! i have no opinion on this. wouldn't you want to pass the same manifest file as in the CLIOptions?

Reply:
I'm not sure what you mean?

Note that current dir is set to the dir that has the specified manifest.